action/protocol/execution/evm/evm.go

```diff
@@ -176,14 +176,23 @@ func newParams(
 		Coinbase:    common.BytesToAddress(blkCtx.Producer.Bytes()),
 		GasLimit:    gasLimit,
 		BlockNumber: new(big.Int).SetUint64(blkCtx.BlockHeight),
-		Time:        new(big.Int).SetInt64(blkCtx.BlockTimeStamp.Unix()),
+		Time:        new(big.Int).SetInt64(blkCtx.BlockTimeStamp.Unix()).Uint64(),
 		Difficulty:  new(big.Int).SetUint64(uint64(50)),
 		BaseFee:     new(big.Int),
 	}
+	if g.IsSumatra(blkCtx.BlockHeight) {
+		// Random opcode (EIP-4399) is not supported on IoTeX
+		zeroHash := common.BytesToHash(nil)
+		context.Random = &zeroHash
+	}
```

Comment:
i recall we now have value = 50? think we can remove this, no need to complicate things with this change.

Reply:
`context.Random = &common.Hash{}` will do